drivers/hmis/spec/requests/hmis/assessments/submit_assessment_spec.rb

```diff
@@ -21,7 +21,7 @@
   let(:c1) { create :hmis_hud_client, data_source: ds1, user: u1 }
   let!(:e1) { create :hmis_hud_enrollment, data_source: ds1, project: p1, client: c1, user: u1, entry_date: 2.weeks.ago }
   let!(:fd1) do
-    ['informationDate', 'fieldOne', 'fieldTwo'].each do |key|
+    ['fieldOne', 'fieldTwo'].each do |key|
       create(:hmis_custom_data_element_definition, key: key, owner_type: Hmis::Hud::CustomAssessment.sti_name, data_source: ds1)
```

Comment:
Comment unrelated to this change:

On a draft PR I added a pattern for creating CDEDs on `after(:create)` of the definition factories. Left a comment [here](https://github.com/greenriver/hmis-warehouse/pull/4607/files#r1713723064) so I could link it. I _think_ it makes sense as a pattern, because if you are using the definition in a test than its most likely you will want the CDEDs to exist, too.

Just wanted to raise the idea. If you want we can wait for discussion until I get around to requesting review on that PR, or you can feel free to comment there.